base/src/main/kotlin/io/spine/string/Strings.kt

```diff
@@ -230,14 +231,35 @@ public fun String.ensurePrefix(prefix: String): String {
 }
 
 /**
- * Returns the simple class name of the type [T].
+ * Returns a simple class name of the type [T] or `<unknown>` if the simple name
+ * is not available.
  */
-public inline  fun <reified T> simply(): String = T::class.simpleName!!
+public inline fun <reified T : Any> simply(): String = T::class.simpleName ?: "<unknown>"
```

Comment:
For reified `T`, a simple name must be present.

I have retained a fall-back option, but not specified in which situration such is possible.